app/models/npc.rb

```diff
@@ -0,0 +1,51 @@
+class Npc < ActiveRecord::Base
+  class ConsequenceLimit < ActiveModel::Validator
+    def validate(npc)
+      if npc.consequences.length > 4
```

Comment:
Also, would it be sensible to take this approach in abstracting out magic numbers in general? For examle:

<pre><code>
  def new_stress_level
    new_level = 2
    case
      when self.level == 1
        new_level = 3
      when self.level.between?(2, 4)
        new_level = 4
      when self.level >= 5
        new_level = 5
    end
    new_level
  end</code></pre>

Would you recommend refactoring the above code using the same pattern as mentioned before?